packages/amplify-e2e-tests/overrides/override-root.ts

```diff
@@ -1,4 +1,7 @@
+function getRandomInt(max) {
+  return Math.floor(Math.random() * max);
+}
 export function override(props: any): void {
-  props.authRole.roleName = 'mockRole';
+  props.authRole.roleName = `mockRole-${getRandomInt(10000)}`;
```

Comment:
nit: I prefer a short uuid for better uniqueness

Reply:
How should I generate it? I think adding any dependencies in this file would be a headache since it is run in generated e2e projects